components/CampaignListItem/index.tsx

```diff
@@ -0,0 +1,121 @@
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined'
+import { Box } from '@mui/material'
+
+const styledmainbox = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '1070px',
+  height: '100%',
+  padding: '10px',
+  backgroundColor: '#EAF9FF',
+}
+
+const styledmainbody = {
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  padding: '0px 20px 0px 20px',
+}
+
+const styledcontentbox = {
+  display: 'flex',
+  height: '300px',
+  backgroundColor: '#D9D9D9',
+}
+
+const styledbottombox = {
+  display: 'flex',
+  flex: 'auto',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '100%',
+  height: '67px',
+  backgroundColor: '#FFFFFF',
+  padding: '0px 20px 0px 20px',
+}
+
+const styledbottomitems = {
+  display: 'flex',
+}
+
+const styledinput = {
+  fontSize: '15',
+  fontWeight: '700',
+}
+
+type Props = {
+  CampaignName: String
+  CampaignDetails: String
+  Donors: Number
+  Amount: Number
+  Country: String
+  Published: Boolean
+  ViewAll: Boolean
+}
+
+const CampaignListItem = ({
+  CampaignName,
+  CampaignDetails,
+  Donors,
+  Amount,
+  Country,
+  Published,
+  ViewAll,
+}: Props) => {
+  return (
+    <Box sx={styledmainbox}>
+      <Box sx={styledmainbody} className="main-body">
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+            flexDirection: 'row',
+          }}
+        >
+          <h3 style={{ color: '#009DD7' }}>{CampaignName}</h3>
+          <ArchiveOutlinedIcon
+            sx={{
+              color: '#009DD7',
+              fontSize: '2.5rem',
+            }}
+          />
+        </div>
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            flexDirection: 'row',
+          }}
+        >
+          <p>{CampaignDetails}</p>
+          <h4
+            style={{
+              color: Published ? '#16A64A' : '#C7043D',
+            }}
```

Comment:
following changes from above, since `published` with now be of type `CampaignStatus` enum, maybe you can have a map instead to store colour.
for eg
```
const campaignStatusColor = {
  [CampaignStatus.PUBLISHED]: '#16A64A',
  [CampaignStatus.DRAFT]: '#C7043D',
  [CampaignStatus.ARCHIVED]: 'purple colour'
} 
```
then this line can be
```
style={{ color: campaignStatusColor[campaignStatus] }}
```

Reply:
Made changes, but i will check what is the hex for purple color